### PR TITLE
Clean Up Existing CIPs

### PR DESCRIPTION
**Pull request by YazzyYaz**
Update existing CIPs to new cip template pre-amble

### CODE REVIEW

**Comment by prestwich** [comment]
LGTM but maybe get review from some of the CIP authors

**Comment by YazzyYaz** [comment]
@nambrot @kevjue @timmoreton please review this PR as it's your proposals. I have updated it with the new CIP template pre-amble.

File: CIPs/cip-0003.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-
-- Date: 2019-09-19
-- Author: @timmoreton
-- Status: Implemented
+---
+cip: 0003
+title: Identity Claims and Metadata 
+author: Tim Moreton <@timmoreton>
+discussions-to: 
+status: Draft
```
Comment (nambrot) [inline CIPs/cip-0003.md:6]: This is technically FINAL i believe

File: CIPs/cip-0004.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
-
-- Date: 2019-10-28
-- Author: @bricewang @timmoreton @yorhodes
-- Status: DRAFT
+---
+cip: 0004 
+title: Protocol Hotfixing Through Governance
+author: Brice Wang <@bricewang>, Tim Moreton <@timmoreton> and Yorke Rhodes <yorhodes>
+discussions-to: 
+status: Draft
```
Comment (nambrot) [inline CIPs/cip-0004.md:6]: I believe this is also FINAL

File: CIPs/cip-0008.md [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
-
-- Date: 2020-05-05
-- Author: @nambrot
-- Status: WIP
+---
+cip: 0008 
+title: Expand Metadata to General Off-Chain Storage
+author: Nam Chu Hoai <@nambrot>
+discussions-to: 
+status: Draft
```
Comment (nambrot) [inline CIPs/cip-0008.md:6]: What would be the appropriate status for implementation Work in Progress, but "accepted" as a standard?

**Review by prestwich — APPROVED** [review]

**Pull request merged**